fed/api.py

```diff
@@ -165,6 +171,7 @@ def init(
     compatible_utils.kv.initialize()
 
     cluster_config = {
+        # TODO: this store party specific conf, not only addresses, give it a better name?
```

Comment:
The discussion can be opened somewhere else but a comment ?

Reply:
removed